eZ/Publish/Core/FieldType/BinaryBase/BinaryBaseStorage/Gateway/DoctrineStorage.php

```diff
@@ -107,6 +107,27 @@ protected function setInsertColumns(QueryBuilder $queryBuilder, VersionInfo $ver
         ;
     }
 
+    /**
+     * @param \Doctrine\DBAL\Query\QueryBuilder $queryBuilder
+     * @param \eZ\Publish\SPI\Persistence\Content\VersionInfo $versionInfo
+     * @param \eZ\Publish\SPI\Persistence\Content\Field $field
+     */
+    protected function setUpdateColumns(QueryBuilder $queryBuilder, VersionInfo $versionInfo, Field $field)
+    {
+        $queryBuilder
+            ->set('contentobject_attribute_id', ':fieldId')
+            ->set('filename', ':filename')
+            ->set('mime_type', ':mimeType')
+            ->set('original_filename', ':originalFilename')
+            ->set('version', ':versionNo')
+            ->setParameter(':fieldId', $field->id, PDO::PARAM_INT)
```

Comment:
Please use for a new code
```suggestion
            ->setParameter(':fieldId', $field->id, ParameterType::INTEGER)
```
to make Doctrine\DBAL 3.0 migration easier in the future.
(not sure if ParameterType is imported in this class, so you may need to do it manually).